src/util.c

```diff
@@ -987,6 +989,20 @@ int fsyncFileDir(const char *filename) {
     return 0;
 }
 
+ /* free OS pages backed by file */
+int invalidatePageCache(int fd, size_t offset, size_t length) {
+#ifdef __linux__
+  int ret = posix_fadvise(fd, offset, length, POSIX_FADV_DONTNEED);
```

Comment:
what abouht `return posix_fadvise(fd, offset, length, POSIX_FADV_DONTNEED);`?

Reply:
Yes, the reason I wrap the ret is I want to normalize the ret to -1 or 0 just as other utility functions,  from man page `posix_fadvise` returns the error number. Or maybe whether -1 is return is not critical.